blog/config/nav.yml

```
@@ -2,8 +2,10 @@ nav:
     - Home: /docs/
     - Tutorial: /docs/getting-started/
```

```suggestion
    - Tutorial: /docs/getting-started/
    - Concepts: /docs/concepts/
```

we were also missing this one too